core/src/smartcontracts/isi/account.rs

```diff
@@ -340,7 +338,7 @@ pub mod isi {
                 .into_iter()
                 .map(|token| token.id);
 
-            state_transaction.world.account(&account_id)?;
+            recognize_account(account_id.clone(), authority, state_transaction)?;
```

Comment:
```suggestion
            recognize_account(&account_id, authority, state_transaction)?;
```

IMO you should move the clone inside the function because the most common execution path will just return